src/ui/pages/BridgeForm/useBridgeTokens.ts

```diff
@@ -0,0 +1,95 @@
+import { useQuery } from '@tanstack/react-query';
+import ky, { HTTPError } from 'ky';
+import type { Chain } from 'src/modules/networks/Chain';
+import { invariant } from 'src/shared/invariant';
+import { DEFI_SDK_TRANSACTIONS_API_URL } from 'src/env/config';
+import { createUrl } from 'src/shared/createUrl';
+
+type Direction = 'input' | 'output' | 'both';
+
+interface Params {
+  inputChain: Chain | null;
+  outputChain: Chain | null;
+  direction: Direction;
+}
+
+interface Response {
+  assets: string[];
+}
+
+interface ParsedHttpError {
+  body: string;
+  detail?: {
+    loc: string[];
+    msg: string;
+    type: string;
+  }[];
+}
+
+export class TransactionsApiError extends HTTPError {
+  static interpretError(parsedError: ParsedHttpError) {
+    if (parsedError.detail && parsedError.detail.length > 0) {
+      const [{ msg }] = parsedError.detail;
```

Comment:
Ok, I see that this was taken from web app. Then let's hope it works correctly :))